### PR TITLE
help: AudioIn: expand description

### PR DESCRIPTION
**Pull request by tiagmoraismorgado**
reviewing description of the ugen

### CODE REVIEW

**Comment by snappizz** [comment]
A better name for this commit would have been "help: AudioIn: expand description."

AudioIn is deprecated and provided for backwards compatibility with SC2 (maybe time to get rid of it soon? 2002 was a long time ago). there's no point in expanding its helpfile.

**Comment by tiagmoraismorgado** [comment]
thanks for the tip :)

**Comment by tiagmoraismorgado** [comment]
we are always learning from each other

File: HelpSource/Classes/AudioIn.schelp [hunk 1]
```diff
@@ -9,10 +9,6 @@ Description::
 
 Grabs audio signal data from any form of transducer connected to an analogue to digital converter in your soundcard, and takes advantage of its DC offset values, to either be used as a controller rate signal, or an audio rate signal, meaning that it can both be used for controlling, for instance, an Oscillator, when passed throughout an audio feature extractor, and interpolated with either envelopes or chaotic functions whatever (controller rate), or to feed data to a buffer or a pfft sound  transformation (audio rate).
 
-note::
```
Comment (snappizz) [inline HelpSource/Classes/AudioIn.schelp]: umm... you removed the deprecation notice?!

**Comment by tiagmoraismorgado** [comment]
@snappizz  I reviewed the suggestions by:

- removing un-relevant changes;
- following the suggestions in relation to removing deprecated text

File: HelpSource/Classes/AudioIn.schelp (hunk 1 above)
Comment (tiagmoraismorgado) [inline HelpSource/Classes/AudioIn.schelp]: sorry i had not read what you said properly

**Comment by snappizz** [comment]
I think we should leave AudioIn untouched and make these changes to SoundIn instead.

**Comment by tiagmoraismorgado** [comment]
ok i will do that. don't close the pull request yet, so that i can copy the changes to SoundIn.ar

**Comment by snappizz** [comment]
Closing a PR doesn't delete your changes. Your branch exists independently of this PR.

**Pull request closed without merging**